modules/installation-osp-provider-network-preparation.adoc

```diff
@@ -46,7 +46,7 @@ Provider networks must be owned by the {rh-openstack} project that is used to cr
 
 * Verify that the provider network can reach the {rh-openstack} metadata service IP address, which is `169.254.169.254` by default.
 +
-Depending on your {rh-openstack} SDN and networking service configuration, you might need to provide the route when you create the subnet. For example:
+Depending on your {rh-openstack} SDN and networking service configuration, you might need to create provide the route when you create the subnet. For example:
```

Comment:
`need to create provide` or `need to create` ? it was `need to create` in https://github.com/openshift/openshift-docs/pull/34461